web-animations-1/Overview.bs

```diff
@@ -708,6 +701,35 @@ animation model to change, but these operations in themselves do not create
 a new [=animation frame=], rather they merely update the current <a>animation
 frame</a>.
 
+### Timeline Phase ### {#timeline-phases}
+
+A [=timeline=] may be in one of four possible 
+<dfn export lt="timeline phase">phases</dfn> for each of which, a non-normative
+description is also provided:
+
+<div class=informative-bg>
+
+:   <dfn export lt="timeline inactive phase">inactive</dfn>
+::  A <a>timeline</a> is considered to be <dfn lt="inactive timeline">inactive
+    </dfn> when its <a>time value</a> is <a>unresolved</a>. This behavior is
+    further defined by each type of timeline.
+:   <dfn export lt="timeline before phase">before</dfn>
+::  The timeline current time is before its start point.
+:   <dfn export lt="timeline active phase">active</dfn>
+::  The timeline current time is after its start point and before its end point.
+    Note: There is a special case for ScrollTimeline that can make the end point
+    inclusive.
+:   <dfn export lt="timeline after phase">after</dfn>
+::  The timeline current time is at or beyond its end point.
+
+</div>
+
+[=timeline phase|Timeline phase=] is used to give additional context to 
+[=timeline current time|current time=]. For example, if a [=timeline=] has a
+delayed start (e.g. a ScrollTimeline startScrollOffset greater than 0),
+[=timeline current time|current time=] of 0 would be ambiguous since it could be
+any amount of time before the start point or it could be at the start time.
```

Comment:
I think this probably belongs in the non-normative section. And it's probably best to avoid referring to specific interface members of the scroll-timeline specification (since that will change). Instead I'd just describe it in slightly more generic terms.